common/src/Microsoft.Azure.IIoT.Core/tests/Utils/LoggingTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.Core.Tests {
+    using Xunit;
+    using static Microsoft.Azure.IIoT.Utils.Logging;
```

Comment:
Great job on the unit test.  But "using static" is not a good pattern to use.   It hides too much and makes the code unmaintainable and difficult to refactor.  Here it should be straightforward to use "Logging." where needed.

Reply:
"using static" is used here only to simplify writing of tests cases. Should it be avoided even in test files?

Reply:
We can debate this as team but I am truly allergic to it :-)

Reply:
Done